### ISSUE TITLE
build error=unused-variable

### ISSUE THREAD

**Issue by EGD** [qTox/qTox#5006]
##### Brief Description

OS: Linux openSUSE Tumbleweed
qTox version: 1.14.0
Commit hash:  5b120c1

Reproducible: Always

##### Steps to reproduce

1. cmake
```
cmake . '-GUnix Makefiles' -DCMAKE_INSTALL_PREFIX:PATH=/usr -DINCLUDE_INSTALL_DIR:PATH=/usr/include -DLIB_INSTALL_DIR:PATH=/usr/lib64 -DSYSCONF_INSTALL_DIR:PATH=/etc -DSHARE_INSTALL_P
REFIX:PATH=/usr/share -DCMAKE_INSTALL_LIBDIR:PATH=/usr/lib64 -DCMAKE_BUILD_TYPE=RelWithDebInfo '-DCMAKE_C_FLAGS=-fmessage-length=0 -grecord-gcc-switches -O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector-strong -funwind-tables -fasynchronous-unwi
nd-tables -fstack-clash-protection -g -DNDEBUG' '-DCMAKE_CXX_FLAGS=-fmessage-length=0 -grecord-gcc-switches -O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -g -DNDEBU
G' '-DCMAKE_Fortran_FLAGS=-fmessage-length=0 -grecord-gcc-switches -O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -g -DNDEBUG' '-DCMAKE_EXE_LINKER_FLAGS=-Wl,--as-nee
ded -Wl,--no-undefined -Wl,-z,now' -DCMAKE_MODULE_LINKER_FLAGS=-Wl,--as-needed '-DCMAKE_SHARED_LINKER_FLAGS=-Wl,--as-needed -Wl,--no-undefined -Wl,-z,now' -DLIB_SUFFIX=64 -DCMAKE_SKIP_RPATH:BOOL=ON -DCMAKE_VERBOSE_MAKEFILE:BOOL=ON -DBUILD_SHAR
ED_LIBS:BOOL=ON -DBUILD_STATIC_LIBS:BOOL=OFF -DCMAKE_COLOR_MAKEFILE:BOOL=OFF -DCMAKE_INSTALL_DO_STRIP:BOOL=OFF -DCMAKE_MODULES_INSTALL_DIR=/usr/share/cmake/Modules
```
2. make

##### Observed Behavior
```
[  118s] /home/abuild/rpmbuild/BUILD/qtox-1.14.0/src/widget/friendwidget.cpp: In member function 'void FriendWidget::onContextMenuCalled(QContextMenuEvent*)':
[  118s] /home/abuild/rpmbuild/BUILD/qtox-1.14.0/src/widget/friendwidget.cpp:144:10: error: unused variable 'friendList' [-Werror=unused-variable]
[  118s]      auto friendList = qobject_cast<FriendListWidget*>(w->parentWidget());
[  118s]           ^~~~~~~~~~
```

**Comment by anthonybilinski** [comment]
It's definitely an unused var, but why are you configuring with so many addition build options instead of just `cmake .`? This doesn't cause an error without making it cause one.

**Comment by EGD** [comment]
> why are you configuring with so many addition build options

i build package for openSUSE and this options defined inside `%cmake` rpm macros

**Comment by anthonybilinski** [comment]
Ok, in that case I will include this fix in the 1.14.1 patch release that will be made today or tomorrow.
